scaling_performance/optimizing_storage.adoc

```diff
@@ -314,32 +312,65 @@ loopback mounted sparse file. To see if your system is using the loopback mode:
+Red Hat strongly recommends using the DeviceMapper storage driver in thin-pool mode for production workloads.
 ====
 
-`Overlay` is also supported for container runtimes use cases as of Red Hat Enterprise Linux
+OverlayFS is also supported for container runtimes use cases as of Red Hat Enterprise Linux
 7.2, and provides faster start up time and page cache sharing, which can
 potentially improve density by reducing overall memory utilization.
 
 [[benefits-of-using-the-overlay-graph-driver]]
-== Benefits of Using the Overlay Graph Driver with SELinux
-
-The default Docker storage configuration on Red Hat Enterprise Linux continues
-to be `DeviceMapper`. While the use of `Overlay` as the container's storage
-technology is under evaluation, moving Red Hat Enterprise Linux to `Overlay` as
-the default in future releases is under consideration. As of Red Hat Enterprise
-Linux 7.2, `Overlay` became a supported graph driver. As of Red Hat Enterprise
-Linux 7.4, SELinux and the `Overlay2` graph driver became a supported
-combination.
-
-The main advantage of the `Overlay` file system is Linux page cache sharing among
-containers sharing an image on the same node. This attribute of `Overlay` leads to
+=== Comparing the overlay vs. overlay2 Graph Drivers 
+
+OverlayFS is a type of union file system. It allows the user to overlay one file system on top of another. 
+Changes are recorded in the upper file system, while the lower file system remains unmodified. 
+This allows multiple users to share a file-system image, such as a container or a DVD-ROM, where the base image is on read-only media. 
+
+OverlayFS layers two directories on a single Linux host and presents them as a single directory. These directories are called layers 
+and the unification process is referred to as a union mount. 
+
+OverlayFS uses one of two graph drivers, *overlay* or *overlay2*. As of Red Hat Enterprise
+Linux 7.2, *overlay*  
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.2_release_notes/technology-preview-file_systems[became a supported graph driver]. 
+As of Red Hat Enterprise Linux 7.4, *overlay2* link:https://access.redhat.com/solutions/2908851[became supported]. SELinux on the docker daemon became supported in 
+Red Hat Enterprise Linux 7.4. See the Red Hat Enterprise Linux release notes for information on using OverlayFS with your version of RHEL, including supportability and usage caveats.
+
+The *overlay2* driver natively supports up to 128 lower OverlayFS layers, where,
+the *overlay* driver works only with a single lower OverlayFS layer. Because of this capability, the *overlay2* driver provides better performance 
+for layer-related Docker commands such as `docker build`, and consumes fewer inodes on the backing filesystem. 
+
+Because the *overlay* driver works only with a single lower OverlayFS layer, multi-layered images cannot be implemented as multiple OverlayFS layers. 
```

Comment:
s/works only/only works
s/multi-layered images cannot be implemented/you cannot implement multi-layered images

Reply:
@kalexand-rh I left the passive in places like this and the above because it wasn't clear to me who or what is doing these things. If it is a user or is it more of a black-box thing. Thoughts?